src/validator.directive.js

```diff
@@ -288,7 +289,7 @@
          */
         if (attrs.validMethod === 'blur') {
           element.bind('blur', function() {
-            var value = ctrl.$viewValue;
+            var value = scope.$eval(attrs.ngModel);
```

Comment:
What's the difference between `ctrl.$viewValue` and `scope.$eval(attrs.ngModel)` in our model.